cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -578,12 +591,13 @@ insertDrepDeRegistration ::
   ReaderT SqlBackend m ()
 insertDrepDeRegistration txId idx cred coin = do
   drepId <- insertCredDrepHash cred
-  void . DB.insertDrepDeRegistration $
-    DB.DrepDeRegistration
-      { DB.drepDeRegistrationTxId = txId
-      , DB.drepDeRegistrationCertIndex = idx
-      , DB.drepDeRegistrationDeposit = Generic.coinToDbLovelace coin
-      , DB.drepDeRegistrationDrepHashId = drepId
+  void . DB.insertDrepRegistration $
+    DB.DrepRegistration
+      { DB.drepRegistrationTxId = txId
+      , DB.drepRegistrationCertIndex = idx
+      , DB.drepRegistrationDeposit = Just (-(fromIntegral $ unCoin coin))
```

Comment:
id the `-` correct here?

Reply:
also looks like the CI is failing when trying to build with cabal

Reply:
Yes, it's negative for deregistrations, meaning a return of the deposit.